aspnetcore/blazor/routing.md

```diff
@@ -97,6 +97,21 @@ The route constraints shown in the following table are available. For the route
 > [!WARNING]
 > Route constraints that verify the URL and are converted to a CLR type (such as `int` or `DateTime`) always use the invariant culture. These constraints assume that the URL is non-localizable.
 
+### Routing with URLs that contain dots
+
+In Blazor server-side apps, the default route in *_Host.cshtml* is `/` (`@page "/"`). A request URL that contains a dot (`.`) isn't matched by the default route because the URL appears to request a file&mdash;the app returns a *404 - Not Found* response for a static file that doesn't exist. To use routes that contain a dot, configure *_Host.cshtml* with the following route template:
```

Comment:
`&mdash;` doesn't MT.  Accept my following comment.

Reply:
In Blazor server-side apps, the default route in *_Host.cshtml* is `/` (`@page "/"`). A request URL that contains a dot (`.`) isn't matched by the default route because the URL appears to request a file. Blazor apps return a *404 - Not Found* response for a static file that doesn't exist. To use routes that contain a dot, configure *_Host.cshtml* with the following route template:

Reply:
I didn't know that ... is that new info from localization? `&mdash;` has been in use for years here. There are **_87_** of them all throughout the docs (~55 files).

Ping @scottaddie, @tdykstra, and @wadepickett to let them know, too.

How do you want to handle them? Eighty-seven makes it challenging to build into a single PR ... find-'n-replace won't be effective in rewriting sentences to get them out.

Reply:
Removing dashes is not a candidate for a script, each instance would require thoughtful rewriting.  I would leave the ones in place that are already there and just be judicious about using them in the future. I have not seen a statement from loc against dashes.

Reply:
I still have quite a few 3.0 topic updates coming up. I'll nix 'um on those as I go. Over time, it should be possible to get them all out of the repo.

Reply:
Wait a minute, some of those are mine, you can leave those in place. 😄

Reply:
I was referring to my assigned 3.0 update topics: The ones that I wrote were the vast majority of that group. If I come to one of your topics, I'll leave them be.